tests/access/test_AccessControl.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+from pathlib import Path
+from starkware.starknet.testing.starknet import Starknet
+from utils import (
+    TestSigner, TRUE, FALSE,
+    assert_event_emitted, assert_revert,
+    get_contract_def, cached_contract
+)
+
+DEFAULT_ADMIN_ROLE = 0
+SOME_OTHER_ROLE = 42
+
+signer = TestSigner(123456789987654321)
+
+
+@pytest.fixture(scope='module')
+def contract_defs():
+    return {
+        Path(key).stem: get_contract_def(key)
+        for key in [
+            'openzeppelin/account/Account.cairo',
+            'tests/mocks/AccessControl.cairo',
+        ]
```

Comment:
interesting, haven't seen this before

Reply:
do you not like it? We can change it for something less fancy

Reply:
i do